File: myblog-server/src/main/java/com/site/blog/service/TimerExeService.java
package com.site.blog.service;

/**
 * @author yanni
 * @date time 2022/6/17 15:57
 * @modified By:
 */
public interface TimerExeService {
    void startTimer(String taskId, String cron, String className);

    void stopTimer(String taskId);
}

File: blog-sever/server-jpa/src/main/kotlin/com/site/blog/service/TimerExeService.kt
package com.site.blog.service

/**
 * @author yanni
 * @date time 2022/6/17 15:57
 * @modified By:
 */
interface TimerExeService {
    fun startTimer(taskId: String?, cron: String?, className: String?)
    fun stopTimer(taskId: String?)

}